common/app/common/commercial/hosted/hardcoded/LeffeHostedPages.scala

```diff
@@ -0,0 +1,156 @@
+package common.commercial.hosted.hardcoded
+
+import common.commercial.hosted._
+import conf.Configuration.site.host
+import conf.Static
+import conf.switches.Switches
+
+object LeffeHostedPages {
+
+  private val willardWiganPageName = "willard-wigan"
+  private val adrienneTreebyPageName = "adrienne-treeby"
+  private val peteLawrencePageName = "pete-lawrence"
+  private val susanDergesPageName = "susan-derges"
+  private val quayBrothersPageName = "quay-brothers"
+
+  private val campaign = HostedCampaign(
+    id = "leffe-rediscover-time",
+    name = "Leffe - rediscover time",
+    owner = "Leffe",
+    logo = HostedLogo(Static("images/commercial/TODO.jpg"))
+  )
+
+  private val cta = HostedCallToAction(
+    url = "TODO",
+    label = "TODO",
+    trackingCode = "TODO",
+    bannerUrl = Static("TODO")
+  )
+
+  private val willardWiganPageWithoutNextPage: HostedVideoPage = HostedVideoPage(
+    campaign,
+    pageUrl = s"$host/advertiser-content/${campaign.id}/$willardWiganPageName",
+    pageName = willardWiganPageName,
+    standfirst = "TODO",
+    video = HostedVideo(
+      mediaId = willardWiganPageName,
+      title = "Willard Wigan",
+      duration = 127,
+      posterUrl = Static("TODO"),
+      srcUrlMp4 = "https://cdn.theguardian.tv/interactive/2016/06/29/160629WillardWigan_V3_2M_H264.mp4",
+      srcUrlWebm = "https://cdn.theguardian.tv/interactive/2016/06/29/160629WillardWigan_V3_2M_vp8.webm",
+      srcUrlOgg = "https://cdn.theguardian.tv/interactive/mp4/1080/2016/06/29/160629WillardWigan_V3_hi.ogv",
+      srcM3u8 = "https://cdn.theguardian.tv/interactive/2016/06/29/HLS/160629WillardWigan_V3.m3u8"
+    ),
+    cta,
+    nextPage = None
+  )
+
+  private val adrienneTreebyPageWithoutNextPage: HostedVideoPage = HostedVideoPage(
+    campaign,
+    pageUrl = s"$host/advertiser-content/${campaign.id}/$adrienneTreebyPageName",
+    pageName = adrienneTreebyPageName,
+    standfirst = "TODO",
+    video = HostedVideo(
+      mediaId = adrienneTreebyPageName,
+      title = "Adrienne Treeby",
+      duration = 116,
+      posterUrl = Static("TODO"),
+      srcUrlMp4 = "https://cdn.theguardian" +
+                  ".tv/interactive/2016/06/29/160629AdrienneTreeby_KP-28311272_h264_mezzanine_2M_H264.mp4",
+      srcUrlWebm = "https://cdn.theguardian" +
+                   ".tv/interactive/2016/06/29/160629AdrienneTreeby_KP-28311272_h264_mezzanine_2M_vp8.webm",
```

Comment:
Yes.  There's a lot of scope for optimisation in this code but I'll do some more refactoring in a separate PR.  There's obviously lots of bits to fill in still as well.